### PR TITLE
[ISSUE-40] Create sample datasets

### PR DESCRIPTION
**Pull request by deusebio**

### CODE REVIEW

File: cgnal/core/datasets/__init__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+DATA_FOLDER = os.path.join(os.path.dirname(os.path.abspath(__file__)), "data")
+
+
+def _create_nyc_dataset(filename: str) -> PandasTimeIndexedDataset:
+    df = pd.read_csv(filename, index_col="Date")
+    return PandasTimeIndexedDataset(features=df.drop("TempM", axis=1), labels=df.TempM)
+
+
+weather_nyc = _create_nyc_dataset(os.path.join(DATA_FOLDER, "weather_nyc_short.csv"))
```
[the remark this answers is not in the document]
Comment (deusebio) [reply cgnal/core/datasets/__init__.py:16]: I don't mind. The only drawback I'm thinking of is that we can't import the data straight away, like 

`from cgnal.core.datasets import weather_nyc`

but it always requires to call the function, but it is not a major point, and I'm fine with swapping to a function

Let's see what also @nromeoarena says

[the remark this answers is not in the document]
Comment (nromeoarena) [reply cgnal/core/datasets/__init__.py:16]: I agree with the @aiola suggestion, better to implement a function call, than defining a global variable.

**Review by nromeoarena — CHANGES_REQUESTED** [review]
apart the comment on the global variable, all seems ok

**Review by aiola — APPROVED** [review]

**Review by nromeoarena — APPROVED** [review]
ok

**Pull request merged**